mozilla/bedrock
# Remove now-404ing link to a webinar from the H2 2021 Transparency Report page

The H2-22 Transparency Report page contained a link to a webinar, but that had started 404ing. This changeset switches it to a TY recording of the page (hat-tip to @reemhamz for finding it!)

## Issue / Bugzilla link

Resolves #11256 

## Testing

* pull locally 
* `make run-prod`
* visit  localhost:8000/en-US/about/policy/transparency/jul-dec-2021/ and confirm there is no link to the webinar
